src/mixins/raster-draw-mixin.js

```diff
@@ -464,6 +464,9 @@ export function rasterDrawMixin(chart) {
       }
       const shapes = drawEngine.sortedShapes
       drawEngine.deleteAllShapes()
+
+      origFilterFunc(Symbol.for("clear"))
```

Comment:
I don't fully understand the fix but it makes sense with regards to how I see this used in other parts of the code. So we don't really need to worry that [Symbol.for](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/for) is not available on IE, as we probably use a shim.